[package] verilator/4.034 fails to build in Windows (Visual Studio 14)

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **verilator/4.034**
  * Operating System+version: **Windows**
  * Compiler+version: **Visual Studio 14**
  * Conan version: **conan 1.30**


### Conan profile 

(Releases fails too)

```
[settings]
arch=x86_64
arch_build=x86_64
build_type=Debug
compiler=Visual Studio
compiler.runtime=MTd
compiler.version=14
os=Windows
os_build=Windows
[options]
[build_requires]
[env]
```



### Logs (Include/Attach if Applicable)
<details><summary>stdout</summary>

```
[HOOK - conan-center.py] pre_export(): [DEPRECATED GLOBAL CPPSTD (KB-H001)] OK
[HOOK - conan-center.py] pre_export(): [REFERENCE LOWERCASE (KB-H002)] OK
[HOOK - conan-center.py] pre_export(): [RECIPE METADATA (KB-H003)] OK
[HOOK - conan-center.py] pre_export(): [HEADER_ONLY, NO COPY SOURCE (KB-H005)] OK
[HOOK - conan-center.py] pre_export(): [FPIC OPTION (KB-H006)] OK
[HOOK - conan-center.py] pre_export(): [VERSION RANGES (KB-H008)] OK
[HOOK - conan-center.py] pre_export(): [RECIPE FOLDER SIZE (KB-H009)] Total recipe size: 19.251953125 KB
[HOOK - conan-center.py] pre_export(): [RECIPE FOLDER SIZE (KB-H009)] OK
[HOOK - conan-center.py] pre_export(): [EXPORT LICENSE (KB-H023)] exports: None
[HOOK - conan-center.py] pre_export(): [EXPORT LICENSE (KB-H023)] exports: patches/**
[HOOK - conan-center.py] pre_export(): [EXPORT LICENSE (KB-H023)] OK
[HOOK - conan-center.py] pre_export(): [TEST PACKAGE FOLDER (KB-H024)] OK
[HOOK - conan-center.py] pre_export(): [META LINES (KB-H025)] OK
[HOOK - conan-center.py] pre_export(): [CONAN CENTER INDEX URL (KB-H027)] OK
[HOOK - conan-center.py] pre_export(): [CMAKE MINIMUM VERSION (KB-H028)] OK
[HOOK - conan-center.py] pre_export(): [TEST PACKAGE - RUN ENVIRONMENT (KB-H029)] OK
[HOOK - conan-center.py] pre_export(): [SYSTEM REQUIREMENTS (KB-H032)] OK
[HOOK - conan-center.py] pre_export(): [CONANDATA.YML FORMAT (KB-H030)] OK
[HOOK - conan-center.py] pre_export(): [TEST PACKAGE - NO IMPORTS() (KB-H034)] OK
[HOOK - conan-center.py] pre_export(): [NO AUTHOR (KB-H037)] OK
[HOOK - conan-center.py] pre_export(): [NO TARGET NAME (KB-H040)] OK
[HOOK - conan-center.py] pre_export(): [NO FINAL ENDLINE (KB-H041)] OK
[HOOK - conan-center.py] pre_export(): [NO REQUIRES.ADD() (KB-H044)] OK
[HOOK - conan-center.py] pre_export(): [DELETE OPTIONS (KB-H045)] OK
[HOOK - conan-center.py] pre_export(): [NO ASCII CHARACTERS (KB-H047)] OK
[HOOK - conan-center.py] pre_export(): [CMAKE VERBOSE MAKEFILE (KB-H046)] OK
[HOOK - conan-center.py] pre_export(): [CMAKE VERSION REQUIRED (KB-H048)] OK
[HOOK - conan-center.py] pre_export(): [CMAKE WINDOWS EXPORT ALL SYMBOLS (KB-H049)] OK
[HOOK - conan-center.py] pre_export(): [DEFAULT OPTIONS AS DICTIONARY (KB-H051)] OK
[HOOK - conan-center.py] pre_export(): [CONFIG.YML HAS NEW VERSION (KB-H052)] OK
[HOOK - conan-center.py] pre_export(): [PRIVATE IMPORTS (KB-H053)] OK
Exporting package recipe
verilator/4.034 exports: File 'conandata.yml' found. Exporting it...
verilator/4.034 exports: Copied 1 '.yml' file: conandata.yml
verilator/4.034 exports_sources: Copied 3 '.patch' files: 0001-split-opt-and-debug-build-install.patch, 0002-msvc-patches.patch, 0003-add-msvc_link-sh.patch
[HOOK - conan-center.py] post_export(): [CONANDATA.YML REDUCE (KB-H031)] Saving conandata.yml: {'sources': {'4.034': {'url': 'https://www.veripool.org/ftp/verilator-4.034.tgz', 'sha256': '54ed7b06ee28b5d21f9d0ee98406d29a508e6124b0d10e54bb32081613ddb80b'}}, 'patches': {'4.034': [{'patch_file': 'patches/0001-split-opt-and-debug-build-install.patch', 'base_path': 'source_subfolder'}, {'patch_file': 'patches/0002-msvc-patches.patch', 'base_path': 'source_subfolder'}, {'patch_file': 'patches/0003-add-msvc_link-sh.patch', 'base_path': 'source_subfolder'}]}}
[HOOK - conan-center.py] post_export(): [CONANDATA.YML REDUCE (KB-H031)] New conandata.yml contents: patches:
  '4.034':
  - base_path: source_subfolder
    patch_file: patches/0001-split-opt-and-debug-build-install.patch
  - base_path: source_subfolder
    patch_file: patches/0002-msvc-patches.patch
  - base_path: source_subfolder
    patch_file: patches/0003-add-msvc_link-sh.patch
sources:
  '4.034':
    sha256: 54ed7b06ee28b5d21f9d0ee98406d29a508e6124b0d10e54bb32081613ddb80b
    url: https://www.veripool.org/ftp/verilator-4.034.tgz

[HOOK - conan-center.py] post_export(): [CONANDATA.YML REDUCE (KB-H031)] OK
[HOOK - conan-center.py] post_export(): [DEFAULT SHARED OPTION VALUE (KB-H050)] OK
verilator/4.034: The stored package has not changed
verilator/4.034: Using the exported files summary hash as the recipe revision: b6b71a7d65e36bbb0fed39385f6bad36 
verilator/4.034: Exported revision: b6b71a7d65e36bbb0fed39385f6bad36
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Debug
compiler=Visual Studio
compiler.runtime=MTd
compiler.version=14
os=Windows
os_build=Windows
[options]
[build_requires]
[env]

verilator/4.034: Forced build from source
verilator/4.034 (test package): Installing package
Requirements
    dirent/1.23.2 from 'central' - Cache
    strawberryperl/5.30.0.1 from 'central' - Cache
    systemc/2.3.3 from 'central' - Cache
    verilator/4.034 from local cache - Cache
Packages
    dirent/1.23.2:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    strawberryperl/5.30.0.1:456f15897172eef340fcbac8a70811f2beb26a93 - Cache
    systemc/2.3.3:3adaddece7169a57056b3606107c1b851e718581 - Cache
    verilator/4.034:41ffc3236a555955cb702fb99b91c413bcb231e1 - Build
Build requirements
    autoconf/2.69 from 'central' - Cache
    automake/1.16.2 from 'central' - Cache
    m4/1.4.18 from 'central' - Cache
    msys2/20190524 from 'central' - Cache
    winflexbison/2.5.22 from 'central' - Cache
Build requirements packages
    autoconf/2.69:ca33edce272a279b24f87dc0d4cf5bbdcffbc187 - Cache
    automake/1.16.2:3e48e69237f7f2196164383ef9dedf0f93cbf249 - Cache
    m4/1.4.18:f27dcd2d5e453cb01f9324a959461903ce8116a0 - Cache
    msys2/20190524:3cbd862cb7a3ea47ed1220dc027f344262d5224f - Cache
    winflexbison/2.5.22:f27dcd2d5e453cb01f9324a959461903ce8116a0 - Cache

Installing (downloading, building) binaries...
dirent/1.23.2: Already installed!
[HOOK - conan-center.py] post_package_info(): [CMAKE FILE NOT IN BUILD FOLDERS (KB-H019)] OK
m4/1.4.18: Already installed!
m4/1.4.18: Appending PATH environment variable: C:\rootfolder\.conan\data\m4\1.4.18\_\_\package\f27dcd2d5e453cb01f9324a959461903ce8116a0\bin
m4/1.4.18: Setting M4 environment variable: C:/rootfolder/.conan/data/m4/1.4.18/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin/m4.exe
[HOOK - conan-center.py] post_package_info(): [CMAKE FILE NOT IN BUILD FOLDERS (KB-H019)] OK
msys2/20190524: Already installed!
msys2/20190524: Creating MSYS_ROOT env var : C:\rootfolder/s\384d31\1\bin
msys2/20190524: Creating MSYS_BIN env var : C:\rootfolder/s\384d31\1\bin\usr\bin
msys2/20190524: Appending PATH env var with : C:\rootfolder/s\384d31\1\bin\usr\bin
[HOOK - conan-center.py] post_package_info(): [CMAKE FILE NOT IN BUILD FOLDERS (KB-H019)] OK
strawberryperl/5.30.0.1: Already installed!
strawberryperl/5.30.0.1: Appending PATH environment variable: C:\rootfolder/s\1542ba\1\bin
[HOOK - conan-center.py] post_package_info(): [CMAKE FILE NOT IN BUILD FOLDERS (KB-H019)] OK
systemc/2.3.3: Already installed!
[HOOK - conan-center.py] post_package_info(): [CMAKE FILE NOT IN BUILD FOLDERS (KB-H019)] OK
winflexbison/2.5.22: Already installed!
winflexbison/2.5.22: Appending PATH environment variable: C:\rootfolder\.conan\data\winflexbison\2.5.22\_\_\package\f27dcd2d5e453cb01f9324a959461903ce8116a0\bin
winflexbison/2.5.22: Setting LEX environment variable: C:/rootfolder/.conan/data/winflexbison/2.5.22/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin/win_flex
winflexbison/2.5.22: Setting YACC environment variable: C:/rootfolder/.conan/data/winflexbison/2.5.22/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin/win_bison -y
[HOOK - conan-center.py] post_package_info(): [CMAKE FILE NOT IN BUILD FOLDERS (KB-H019)] OK
autoconf/2.69: Already installed!
autoconf/2.69: Appending PATH env var with : C:\rootfolder\.conan\data\autoconf\2.69\_\_\package\ca33edce272a279b24f87dc0d4cf5bbdcffbc187\bin
autoconf/2.69: Setting AC_MACRODIR to C:\rootfolder\.conan\data\autoconf\2.69\_\_\package\ca33edce272a279b24f87dc0d4cf5bbdcffbc187\bin\share\autoconf
autoconf/2.69: Setting AUTOCONF to /rootfolder/.conan/data/autoconf/2.69/_/_/package/ca33edce272a279b24f87dc0d4cf5bbdcffbc187/bin/autoconf.exe
autoconf/2.69: Setting AUTORECONF to /rootfolder/.conan/data/autoconf/2.69/_/_/package/ca33edce272a279b24f87dc0d4cf5bbdcffbc187/bin/autoreconf.exe
autoconf/2.69: Setting AUTOHEADER to /rootfolder/.conan/data/autoconf/2.69/_/_/package/ca33edce272a279b24f87dc0d4cf5bbdcffbc187/bin/autoheader.exe
autoconf/2.69: Setting AUTOM4TE to /rootfolder/.conan/data/autoconf/2.69/_/_/package/ca33edce272a279b24f87dc0d4cf5bbdcffbc187/bin/autom4te.exe
autoconf/2.69: Setting AUTOM4TE_PERLLIBDIR to C:\rootfolder\.conan\data\autoconf\2.69\_\_\package\ca33edce272a279b24f87dc0d4cf5bbdcffbc187\bin\share\autoconf
[HOOK - conan-center.py] post_package_info(): [CMAKE FILE NOT IN BUILD FOLDERS (KB-H019)] OK
automake/1.16.2: Already installed!
automake/1.16.2: Appending PATH env var with : C:\rootfolder\.conan\data\automake\1.16.2\_\_\package\3e48e69237f7f2196164383ef9dedf0f93cbf249\bin
automake/1.16.2: Setting ACLOCAL to /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/aclocal.exe
automake/1.16.2: Setting AUTOMAKE_DATADIR to /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share
automake/1.16.2: Setting AUTOMAKE_LIBDIR to /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16
automake/1.16.2: Setting AUTOMAKE_PERLLIBDIR to /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16
automake/1.16.2: Setting AUTOMAKE to /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/automake.exe
automake/1.16.2: Append M4 include directories to AUTOMAKE_CONAN_INCLUDES environment variable
[HOOK - conan-center.py] post_package_info(): [CMAKE FILE NOT IN BUILD FOLDERS (KB-H019)] OK
verilator/4.034: Applying build-requirement: msys2/20190524
verilator/4.034: Applying build-requirement: automake/1.16.2
verilator/4.034: Applying build-requirement: winflexbison/2.5.22
verilator/4.034: Applying build-requirement: dirent/1.23.2
verilator/4.034: Applying build-requirement: autoconf/2.69
verilator/4.034: Applying build-requirement: m4/1.4.18
verilator/4.034: Copying sources to build folder
verilator/4.034: Building your package in C:\rootfolder\.conan\data\verilator\4.034\_\_\build\41ffc3236a555955cb702fb99b91c413bcb231e1
verilator/4.034: Generator txt created conanbuildinfo.txt
[HOOK - conan-center.py] pre_build(): [FPIC MANAGEMENT (KB-H007)] 'fPIC' option not found
[HOOK - conan-center.py] pre_build(): [FPIC MANAGEMENT (KB-H007)] OK
verilator/4.034: Calling build()
verilator/4.034: run_in_windows_bash: C:\rootfolder\s\384d31\1\bin\usr\bin\bash.exe --login -c ^"cd \^"/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1\^" ^&^& PATH=\^"/rootfolder/s/384d31/1/bin/usr/bin:/rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin:/rootfolder/.conan/data/winflexbison/2.5.22/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin:/rootfolder/s/1542ba/1/bin:/rootfolder/.conan/data/autoconf/2.69/_/_/package/ca33edce272a279b24f87dc0d4cf5bbdcffbc187/bin:/rootfolder/.conan/data/m4/1.4.18/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin:$PATH\^" ^&^& /rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/source_subfolder/configure --help ^"

----Running------
> C:\rootfolder\s\384d31\1\bin\usr\bin\bash.exe --login -c ^"cd \^"/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1\^" ^&^& PATH=\^"/rootfolder/s/384d31/1/bin/usr/bin:/rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin:/rootfolder/.conan/data/winflexbison/2.5.22/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin:/rootfolder/s/1542ba/1/bin:/rootfolder/.conan/data/autoconf/2.69/_/_/package/ca33edce272a279b24f87dc0d4cf5bbdcffbc187/bin:/rootfolder/.conan/data/m4/1.4.18/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin:$PATH\^" ^&^& /rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/source_subfolder/configure --help ^"
-----------------
`configure' configures Verilator 4.034 2020-05-03 to adapt to many kinds of systems.

Usage: /rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/source_subfolder/configure [OPTION]... [VAR=VALUE]...

To assign environment variables (e.g., CC, CFLAGS...), specify them as
VAR=VALUE.  See below for descriptions of some of the useful variables.

Defaults for the options are specified in brackets.

Configuration:
  -h, --help              display this help and exit
      --help=short        display options specific to this package
      --help=recursive    display the short help of all the included packages
  -V, --version           display version information and exit
  -q, --quiet, --silent   do not print `checking ...' messages
      --cache-file=FILE   cache test results in FILE [disabled]
  -C, --config-cache      alias for `--cache-file=config.cache'
  -n, --no-create         do not create output files
      --srcdir=DIR        find the sources in DIR [configure dir or `..']

Installation directories:
  --prefix=PREFIX         install architecture-independent files in PREFIX
                          [/usr/local]
  --exec-prefix=EPREFIX   install architecture-dependent files in EPREFIX
                          [PREFIX]

By default, `make install' will install all the files in
`/usr/local/bin', `/usr/local/lib' etc.  You can specify
an installation prefix other than `/usr/local' using `--prefix',
for instance `--prefix=$HOME'.

For better control, use the options below.

Fine tuning of the installation directories:
  --bindir=DIR            user executables [EPREFIX/bin]
  --sbindir=DIR           system admin executables [EPREFIX/sbin]
  --libexecdir=DIR        program executables [EPREFIX/libexec]
  --sysconfdir=DIR        read-only single-machine data [PREFIX/etc]
  --sharedstatedir=DIR    modifiable architecture-independent data [PREFIX/com]
  --localstatedir=DIR     modifiable single-machine data [PREFIX/var]
  --runstatedir=DIR       modifiable per-process data [LOCALSTATEDIR/run]
  --libdir=DIR            object code libraries [EPREFIX/lib]
  --includedir=DIR        C header files [PREFIX/include]
  --oldincludedir=DIR     C header files for non-gcc [/usr/include]
  --datarootdir=DIR       read-only arch.-independent data root [PREFIX/share]
  --datadir=DIR           read-only architecture-independent data [DATAROOTDIR]
  --infodir=DIR           info documentation [DATAROOTDIR/info]
  --localedir=DIR         locale-dependent data [DATAROOTDIR/locale]
  --mandir=DIR            man documentation [DATAROOTDIR/man]
  --docdir=DIR            documentation root [DATAROOTDIR/doc/verilator]
  --htmldir=DIR           html documentation [DOCDIR]
  --dvidir=DIR            dvi documentation [DOCDIR]
  --pdfdir=DIR            pdf documentation [DOCDIR]
  --psdir=DIR             ps documentation [DOCDIR]

Optional Features:
  --disable-option-checking  ignore unrecognized --enable/--with options
  --disable-FEATURE       do not include FEATURE (same as --enable-FEATURE=no)
  --enable-FEATURE[=ARG]  include FEATURE [ARG=yes]
  --disable-dependency-tracking
                          ignored
  --enable-maintainer-mode
                          ignored
  --disable-silent-rules  ignored
  --disable-partial-static
                          By default, for Verilation peformance, Verilator is
                          linked against some of its dependencies statically.
                          Use this to link the Verilator binary fully
                          dynamically.
  --enable-tcmalloc       Use libtcmalloc_minimal for faster dynamic memory
                          management in Verilator binary[default=check]
  --disable-defenv        disable using some hardcoded data paths extracted
                          from some default environment variables (the default
                          is to use hardcoded paths) in Verilator binary
  --enable-ccwarn         enable showing and stopping on compilation warnings
                          in Verilator binrary and Verilated makefiles
  --enable-longtests      enable running long developer tests
  --enable-prec11         enable pre-C++11 compilers for Verilator binary and
                          Verilated makefiles

Some influential environment variables:
  CC          C compiler command
  CFLAGS      C compiler flags
  LDFLAGS     linker flags, e.g. -L<lib dir> if you have libraries in a
              nonstandard directory <lib dir>
  LIBS        libraries to pass to the linker, e.g. -l<library>
  CPPFLAGS    (Objective) C/C++ preprocessor flags, e.g. -I<include dir> if
              you have headers in a nonstandard directory <include dir>
  CXX         C++ compiler command
  CXXFLAGS    C++ compiler flags
  CXXCPP      C++ preprocessor

Use these variables to override the choices made by `configure' or to help
it to find libraries and programs with nonstandard names/locations.

Report bugs to <https://verilator.org>.
Verilator home page: <https://verilator.org>.
verilator/4.034: Calling:
 > /rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/source_subfolder/configure --datarootdir=/rootfolder/.conan/data/verilator/4.034/_/_/package/41ffc3236a555955cb702fb99b91c413bcb231e1/bin/share --prefix=C:/rootfolder/.conan/data/verilator/4.034/_/_/package/41ffc3236a555955cb702fb99b91c413bcb231e1 
verilator/4.034: run_in_windows_bash: C:\rootfolder\s\384d31\1\bin\usr\bin\bash.exe --login -c ^"cd \^"/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1\^" ^&^& PATH=\^"/rootfolder/s/384d31/1/bin/usr/bin:/rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin:/rootfolder/.conan/data/winflexbison/2.5.22/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin:/rootfolder/s/1542ba/1/bin:/rootfolder/.conan/data/autoconf/2.69/_/_/package/ca33edce272a279b24f87dc0d4cf5bbdcffbc187/bin:/rootfolder/.conan/data/m4/1.4.18/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin:$PATH\^" ^&^& /rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/source_subfolder/configure --datarootdir=/rootfolder/.conan/data/verilator/4.034/_/_/package/41ffc3236a555955cb702fb99b91c413bcb231e1/bin/share --prefix=C:/rootfolder/.conan/data/verilator/4.034/_/_/package/41ffc3236a555955cb702fb99b91c413bcb231e1  ^"

----Running------
> C:\rootfolder\s\384d31\1\bin\usr\bin\bash.exe --login -c ^"cd \^"/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1\^" ^&^& PATH=\^"/rootfolder/s/384d31/1/bin/usr/bin:/rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin:/rootfolder/.conan/data/winflexbison/2.5.22/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin:/rootfolder/s/1542ba/1/bin:/rootfolder/.conan/data/autoconf/2.69/_/_/package/ca33edce272a279b24f87dc0d4cf5bbdcffbc187/bin:/rootfolder/.conan/data/m4/1.4.18/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin:$PATH\^" ^&^& /rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/source_subfolder/configure --datarootdir=/rootfolder/.conan/data/verilator/4.034/_/_/package/41ffc3236a555955cb702fb99b91c413bcb231e1/bin/share --prefix=C:/rootfolder/.conan/data/verilator/4.034/_/_/package/41ffc3236a555955cb702fb99b91c413bcb231e1  ^"
-----------------
configure: loading site script /mingw64/etc/config.site
configuring for Verilator 4.034 2020-05-03
checking whether to perform partial static linking of Verilator binary... yes
checking whether to use tcmalloc... check
checking whether to use hardcoded paths... yes
checking whether to show and stop on compilation warnings... no
checking whether to run long tests... no
checking whether allow pre-C++11... no
checking for gcc... /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo
checking whether the C compiler works... yes
checking for C compiler default output file name... conftest.exe
checking for suffix of executables... .exe
checking whether we are cross compiling... no
checking for suffix of object files... obj
checking whether we are using the GNU C compiler... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -g... no
checking for /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo option to accept ISO C89... none needed
checking whether we are using the GNU C++ compiler... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -g... no
checking for a BSD-compatible install... /usr/bin/install -c
compiler is /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo --version = 
checking that C++ compiler can compile simple program... yes
checking for perl... /rootfolder/s/384d31/1/bin/usr/bin/perl
checking for flex... C:/rootfolder/.conan/data/winflexbison/2.5.22/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin/win_flex
checking for bison... C:/rootfolder/.conan/data/winflexbison/2.5.22/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin/win_bison -y
checking for ccache... no
checking how to run the C++ preprocessor... /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo -E
checking for grep that handles long lines and -e... /rootfolder/s/384d31/1/bin/usr/bin/grep
checking for egrep... /rootfolder/s/384d31/1/bin/usr/bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... no
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... no
checking for size_t... yes
checking for size_t... (cached) yes
checking for inline... inline
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -std=gnu++14... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -std=gnu++11... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -std=gnu++03... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -std=c++14... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -std=c++11... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -std=c++03... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -std=c++03... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -std=c++11... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -std=c++14... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -std=gnu++03... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -std=gnu++11... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -std=gnu++14... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wextra... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wfloat-conversion... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wlogical-op... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wthread-safety... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Qunused-arguments... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -faligned-new... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-unused-parameter... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-shadow... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-char-subscripts... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-null-conversion... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-parentheses-equality... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-unused... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -faligned-new... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -fbracket-depth=4096... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -fcf-protection=none... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -mno-cet... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Qunused-arguments... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-bool-operation... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-tautological-bitwise-compare... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-parentheses-equality... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-sign-compare... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-uninitialized... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-unused-but-set-variable... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-unused-parameter... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-unused-variable... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo accepts -Wno-shadow... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo linker accepts -mt... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo linker accepts -pthread... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo linker accepts -lpthread... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo linker accepts -latomic... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo linker accepts -static-libgcc... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo linker accepts -static-libstdc++... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo linker accepts -Xlinker -gc-sections... yes
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo linker accepts -lpthread... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo linker accepts -lbcrypt... yes
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo linker accepts -lpsapi... yes
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo linker accepts -l:libtcmalloc_minimal.a... no
checking whether /rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin/share/automake-1.16/compile cl -nologo supports Verilated threads... yes
checking for struct stat.st_mtim.tv_nsec... no
configure: creating ./config.status
config.status: creating Makefile
config.status: creating docs/Makefile
config.status: creating src/Makefile
config.status: creating src/Makefile_obj
config.status: creating include/verilated.mk
config.status: creating include/verilated_config.h
config.status: creating verilator.pc
config.status: creating verilator-config.cmake
config.status: creating verilator-config-version.cmake
config.status: creating src/config_build.h

Now type 'make' (or sometimes 'gmake') to build Verilator.

verilator/4.034: run_in_windows_bash: C:\rootfolder\s\384d31\1\bin\usr\bin\bash.exe --login -c ^"cd \^"/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1\^" ^&^& PATH=\^"/rootfolder/s/384d31/1/bin/usr/bin:/rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin:/rootfolder/.conan/data/winflexbison/2.5.22/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin:/rootfolder/s/1542ba/1/bin:/rootfolder/.conan/data/autoconf/2.69/_/_/package/ca33edce272a279b24f87dc0d4cf5bbdcffbc187/bin:/rootfolder/.conan/data/m4/1.4.18/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin:$PATH\^" ^&^& make CFG_WITH_DEFENV=false SRC_TARGET=dbg DEBUG=1 PROGLINK=/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/source_subfolder/msvc_link.sh -j1 ^"

----Running------
> C:\rootfolder\s\384d31\1\bin\usr\bin\bash.exe --login -c ^"cd \^"/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1\^" ^&^& PATH=\^"/rootfolder/s/384d31/1/bin/usr/bin:/rootfolder/.conan/data/automake/1.16.2/_/_/package/3e48e69237f7f2196164383ef9dedf0f93cbf249/bin:/rootfolder/.conan/data/winflexbison/2.5.22/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin:/rootfolder/s/1542ba/1/bin:/rootfolder/.conan/data/autoconf/2.69/_/_/package/ca33edce272a279b24f87dc0d4cf5bbdcffbc187/bin:/rootfolder/.conan/data/m4/1.4.18/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin:$PATH\^" ^&^& make CFG_WITH_DEFENV=false SRC_TARGET=dbg DEBUG=1 PROGLINK=/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/source_subfolder/msvc_link.sh -j1 ^"
-----------------
------------------------------------------------------------
making verilator in src
make -C src dbg 
make[1]: Entering directory '/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/src'
mkdir -p obj_dbg
mkdir -p ../bin
make -C obj_dbg -j 1  TGT=../../bin/verilator_bin_dbg VL_DEBUG=1 -f ../Makefile_obj serial
make[2]: Entering directory '/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/src/obj_dbg'
/rootfolder/s/384d31/1/bin/usr/bin/perl /rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/source_subfolder/src/astgen -I/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/source_subfolder/src  --classes
If you get errors from verilog.y below, try upgrading bison to version 1.875 or newer.
/rootfolder/s/384d31/1/bin/usr/bin/perl /rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/source_subfolder/src/bisonpre --yacc C:/rootfolder/.conan/data/winflexbison/2.5.22/_/_/package/f27dcd2d5e453cb01f9324a959461903ce8116a0/bin/win_bison -y -d -v -o V3ParseBison.c /rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/source_subfolder/src/verilog.y
make[2]: Leaving directory '/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/src/obj_dbg'
make[1]: Leaving directory '/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/src'
verilator/4.034: 

JSON file created at 'C:\rootfolder/4/d24b09b3-6b9a-4bdd-adc5-ff6737ac685c/_create.json'

```

</details>



<details><summary>stderr</summary>

```
verilator/4.034: WARN: Build folder is dirty, removing it: C:\rootfolder\.conan\data\verilator\4.034\_\_\build\41ffc3236a555955cb702fb99b91c413bcb231e1
cl : Command line warning D9002 : ignoring unknown option '--version'
cl : Command line error D8003 : missing source filename
Unknown option: y
%Error: Bad usage, try 'bisonpre --help'
make[2]: *** [../Makefile_obj:325: V3ParseBison.c] Error 255
make[1]: *** [Makefile:65: ../bin/verilator_bin_dbg] Error 2
make: *** [Makefile:221: verilator_exe] Error 2
verilator/4.034: ERROR: Package '41ffc3236a555955cb702fb99b91c413bcb231e1' build failed
verilator/4.034: WARN: Build folder C:\rootfolder\.conan\data\verilator\4.034\_\_\build\41ffc3236a555955cb702fb99b91c413bcb231e1
ERROR: verilator/4.034: Error in build() method, line 108
	autotools.make(args=self._make_args)
	ConanException: Error 2 while executing make CFG_WITH_DEFENV=false SRC_TARGET=dbg DEBUG=1 PROGLINK=/rootfolder/.conan/data/verilator/4.034/_/_/build/41ffc3236a555955cb702fb99b91c413bcb231e1/source_subfolder/msvc_link.sh -j1


```

</details>

--- a/recipes/verilator/all/conanfile.py
+++ b/recipes/verilator/all/conanfile.py
@@ -73,7 +73,12 @@ class VerilatorConan(ConanFile):
         conf_args = [
             "--datarootdir={}/bin/share".format(tools.unix_path(self.package_folder)),
         ]
-        self._autotools.configure(args=conf_args, configure_dir=os.path.join(self.build_folder, self._source_subfolder))
+        yacc = tools.get_env("YACC")
+        if yacc:
+            if yacc.endswith(" -y"):
+                yacc = yacc[:-3]
+        with tools.environment_append({"YACC": yacc}):
+            self._autotools.configure(args=conf_args, configure_dir=os.path.join(self.build_folder, self._source_subfolder))
         return self._autotools
 
     @property

--- a/recipes/verilator/all/test_package/CMakeLists.txt
+++ b/recipes/verilator/all/test_package/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.1)
 project(PackageTest CXX)
 
 include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")

